mayhem/fuzz.go

```diff
@@ -0,0 +1,15 @@
+package fuzz
```

Comment:
I believe you can take your fuzzed data directly as a string without having to cast it.

Reply:
Could you provide more detail about this? Go throws an error when I pass the byte array directly to the IsValidImageID function and the go-fuzz Fuzz function needs to pass bytes or it won't be recognized as a fuzz function.

Reply:
I was thinking of structure aware approaches but I think this is fine:
https://adalogics.com/blog/structure-aware-go-fuzzing-complex-types